# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmBaselineCompile.h
# Commit: 48d5f5918e01
# Full Hash: 48d5f5918e01a7a292ba4cca6e6ffffa055b986d
# Author: Dmitry Bezhetskov <dbezhetskov@igalia.com>
# Date: 2020-10-31 21:05:17
# Regressor Bug: 1639153
# File Overlap Count: 1
# Description:
#   Bug 1639153 - Part 1: Reserve two slots after stack arguments for the future tls preservation. r=lth
#   
#   We are going to remove Frame::tls and support trampolines for indirect calls, so we need to get rid of using Frame::tls.
#   In this and the followup patches I will iteratively remove all dependencies of Frame::tls and remove it eventually.
#   
# ==============================================================================

diff -r 5995c2f5c003 -r 48d5f5918e01 js/src/wasm/WasmBaselineCompile.h
--- a/js/src/wasm/WasmBaselineCompile.h	Sat Oct 31 12:37:34 2020 +0200
+++ b/js/src/wasm/WasmBaselineCompile.h	Sat Oct 31 09:56:04 2020 +0000
@@ -41,7 +41,7 @@
 
   const ValTypeVector& locals_;
   const ArgTypeVector& args_;
-  jit::ABIArgIter<ArgTypeVector> argsIter_;
+  jit::WasmABIArgIter<ArgTypeVector> argsIter_;
   size_t index_;
   int32_t frameSize_;
   int32_t nextFrameSize_;